content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md

```diff
@@ -33,6 +33,49 @@ You will need to have the following tools installed:
 - `cosign`（[安装指南](https://docs.sigstore.dev/cosign/installation/)）
 - `curl`（通常由你的操作系统提供）
 
+<!-- 
+## Verifying binary signatures
+
+The Kubernetes release process signs all binary artifacts (tarballs, SPDX files,
+standalone binaries) by using cosign's keyless signing. To verify a particular
+binary, retrieve it together with its signature and certificate: -->
+
+## 验证二进制签名 {#verifying-binary-signatures}
+
+Kubernetes 发布过程使用 cosign 的无密钥签名对所有二进制制品（tarballs、SPDX 文件、独立二进制文件）进行了签名。要验证特定的二进制文件，在获取这个二进制文件的时候，可以配套获取签名和证书的信息，参考下面的示例：
```

Comment:
```suggestion
Kubernetes 发布过程对所有二进制工件（压缩包、SPDX 文件、 独立的二进制文件）通过使用 cosign 的无密钥签名。
要验证一个特定的二进制文件，连同其签名和证书一起获取：
```

Reply:
Thank you for taking the time to review my pull request and for providing these suggestions. I have made the requested changes to the code . Please let me know if you have any further questions or concerns. Thank you again for your help!

Reply:
Thank you for taking the time to review my pull request and for providing these suggestions. I have made the requested changes to the code . Please let me know if you have any further questions or concerns. Thank you again for your help!